src/app/veilederverktoy/begrunnelseform/begrunnelse-textarea.tsx

```diff
@@ -2,7 +2,7 @@ import FormikTekstArea from '../../components/formik/formik-textarea';
 import React from 'react';
 import { validerBeskrivelse, validerBeskrivelse500TegnFeldt } from '../../utils/formik-validation';
 
-export function BegrunnelseTextArea(props: { tekstariaLabel: string; maxLength?: number }) {
+export function BegrunnelseTextArea(props: { tekstariaLabel: string; maxLength?: number; content?: string }) {
```

Comment:
Hvorfor trenger denne en ekstra prop ?

Reply:
Ser ikke ut som den er brukt ?

Reply:
Leftovers, er vel litt for vant til at IntelliJ advarer mot ubrukte args... :/